mambaTests/Util Tests/MambaUtilTests.swift

```diff
@@ -98,9 +98,9 @@ class MambaUtilTests: XCTestCase {
     
     func testConvenienceString() {
         let desc = String(describing: PantosTag.EXTINF)
-        XCTAssert(PantosTag(rawValue: desc) == PantosTag.EXTINF, "Round trip HLSTagDescriptor->String->HLSTagDescriptor did not work")
+        XCTAssert(PantosTag(rawValue: desc) == PantosTag.EXTINF, "Round trip TagDescriptor->String->TagDescriptor did not work")
```

Comment:
`PlaylistTagDescriptor`? This style is used in other test files. I won't comment on those.

Reply:
I think I'm just going to leave this one since it's a unit test and is clear, i think.